build/deploy/generate-deploy-examples.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+set -e
+
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
+MANIFEST_FILE=../../deploy/examples/multus-validation.yaml
+SED=${SCRIPT_ROOT}/../../build/sed-in-place
+run_gen() {
+if [[ ${SKIP_GEN} = true ]]; then
+  echo "SKIP_GEN is set, skipping manifest  generation"
+  exit
+fi
+OUT_FILE="${SCRIPT_ROOT}/${MANIFEST_FILE}"
+TMP_FILE="$(mktemp)"
+
+START_MARKER='# THE BELOW HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+END_MARKER='# THE ABOVE HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+
+go run ${SCRIPT_ROOT}/../../cmd/rook/*.go multus validation run --help |  sed -e '/^\s*$/d' | awk '{print "# "$0}' > ${TMP_FILE}
```

Comment:
It looks like this is removing newlines from the `--help` output, but that makes it less readable

```suggestion
go run ${SCRIPT_ROOT}/../../cmd/rook/*.go multus validation run --help | awk '{print "# "$0}' > ${TMP_FILE}
```

Reply:
@BlaineEXE  wrote:
> It looks like this is removing newlines from the `--help` output,

correct.

>  but that makes it less readable

oh ok. I agree, but I  added  this after @subhamkrai pointed out that the commented out blank lines caused yamllint errors. this thixed those.